packages/frontend-api/apiary.apib

```diff
@@ -2573,3 +2573,339 @@ Returns brand filtered using UUID
             }
         }
     }
+
+## List of promoted products [/graphql{?topProducts}]
+
+### POST [POST]
+
+Returns complete list of promoted (top) products
+
+- Request (application/json)
+
+    {
+        topProducts {
+            uuid
+            name
+            shortDescription
+            link
+            unit {
+                name
+            }
+            isUsingStock
+            availability {
+                name
+            }
+            stockQuantity
+            categories {
+                name
+            }
+            flags {
+                name
+                rgbColor
+            }
+            price {
+                priceWithVat
+                priceWithoutVat
+                vatAmount
+            }
+            images {
+                type
+                position
+                size
+                url
+                width
+                height
+            }
+            brand {
+                name
+            }
+        }
+    }
+
+- Response 200 (application/json; charset=utf-8)
+
+    {
+        "data": {
+            "topProducts": [
```

Comment:
please, rename to something more fitting, say "promotedProducts"
also, I would like to see similar english H1 title on admin page, so it is obvious, whats what.

Reply:
fixed, thanks!